### PR TITLE
Modified Kafka settings in strimzi example, noted this is purely for …

### PR DESCRIPTION
**Pull request by RayRoestenburg**
…testing.

### What changes were proposed in this pull request?

Modified strimzi Kafka cluster configuration. 

### Why are the changes needed?

Users quickly run into Kafka running out of space.

### Does this PR introduce any user-facing change?

docs

### CODE REVIEW

File: docs/docs-source/docs/modules/administration/pages/how-to-install-and-use-strimzi.adoc [hunk 1]
```diff
@@ -62,8 +64,9 @@ spec:
     config:
       auto.create.topics.enable: false
       log.message.format.version: "2.3"
-      log.retention.bytes: 112197632
-      log.retention.hours: -1
+      log.retention.bytes: 1073741824
+      log.retention.hours: 1
```
Comment (huntc) [inline docs/docs-source/docs/modules/administration/pages/how-to-install-and-use-strimzi.adoc:68]: I think this is good - at least there is a limit on retention now.

File: docs/docs-source/docs/modules/administration/pages/how-to-install-and-use-strimzi.adoc [hunk 2]
```diff
@@ -80,7 +83,7 @@ spec:
     resources: {}
     storage:
       deleteClaim: false
-      size: 1Gi
+      size: 100Gi
```
Comment (huntc) [inline docs/docs-source/docs/modules/administration/pages/how-to-install-and-use-strimzi.adoc:86]: My head is in the embedded world still. Crazy numbers! ;-)

**Review by huntc — APPROVED** [review]

**Comment by huntc** [comment]
Actually, can we not use log compaction to achieve the retainment of just he last message?

**Comment by RayRoestenburg** [comment]
Yes we can configure log.compaction , I think that would be ok. It would keep the last message for a key, so it depends on your partitioning logic if that helps in any way. I've added it though.

**Comment by huntc** [comment]
> Yes we can configure log.compaction , I think that would be ok. It would keep the last message for a key, so it depends on your partitioning logic if that helps in any way. I've added it though.

Nice one. That should help a lot.

**Comment by RayRoestenburg** [comment]
I tested the config on GKE, the kafka config in /tmp/strimzi.properties reflects this:
```
kafka.Kafka /tmp/strimzi.properties
```
```
tail -12 /tmp/strimzi.properties
##########
# User provided configuration
##########
auto.create.topics.enable=false
log.cleanup.policy=compact
log.message.format.version=2.3
log.retention.bytes=1073741824
log.retention.check.interval.ms=300000
log.retention.hours=1
offsets.topic.replication.factor=3
transaction.state.log.min.isr=2
transaction.state.log.replication.factor=3
```

**Pull request merged**